pom.xml

```diff
@@ -557,7 +557,7 @@
     </dependency>
     <dependency>
       <groupId>org.testcontainers</groupId>
-      <artifactId>kafka</artifactId>
+      <artifactId>testcontainers</artifactId>
```

Comment:
I would not recommend copying `KafkaContainer` just for the sake of changing a couple of lines. Consider either extending from it, or pre-configuring (one can do `withEnv` on custom containers as well)

Reply:
Also, the implementation seems to be outdated, we dramatically simplified `KafkaContainer` in 1.15.0

Reply:
Thank you! Will take a look when I get home tonight.

Reply:
Hey, nice to see you over here @bsideup !! I tried adding something like this a while ago but never got into playing a lot on creating the connector, so I am glad you added it @sverrehu appreciate a ton! 

It would be awesome to have one as well for the RBAC part.

Reply:
So... I spent some hours making an implementation of SaslPlaintextKafkaContainer that extends KafkaContainer, as per @bsideup's suggestions. I found that KafkaContainer (from 1.15.0-rc2) was not very extends-friendly, at least not for my needs. This implementation is not more readable as I see it, and in addition depends on the inner workings of KafkaContainer not being changed. Maybe it is just me that lacks a proper understanding of Kafka and testcontainers...

Feel free to have a look:

https://github.com/sverrehu/kafka-testcontainers-test/blob/main/src/main/java/no/shhsoft/kafka/DerivedSaslPlaintextKafkaContainer.java

Reply:
As for the RBAC part, @purbon, of course. But as ACLs and RBAC are two separate test profiles in the pom, I tought it OK to split.

Reply:
@sverrehu re RBAC: yes, for sure this was mostly a wish statement if you aim for having a shot at it, is already great to have you helping out with this task. Honestly, it has been a long-pending task to fulfil and I appreciate a ton the help here.

Reply:
@sverrehu re the inheriting, while I do understand where @bsideup is coming up and I agree with him, I do agree with you as well that the current implementation of KafkaContainer it made the extension and especially to have an improved SASL security on it super cumbersome. 
I did a few tries that look like the ones you shared in your previous link, and honestly not that happy with either of that.

For the purpose of here, I would rather go with having "something" useful and we can always come back to improve it. What do you think?

Reply:
@sverrehu thanks for sharing the code! It is indeed not that big of an improvement compared to just copying. I will think about making `KafkaContainer` easier to customize. Meanwhile, you can proceed with the one that extends `GenericContainer`, just make sure that you sync it with the latest version of `KafkaContainer` to get the improvements that we have made 👍

Reply:
> @sverrehu re the inheriting, while I do understand where @bsideup is coming up and I agree with him, I do agree with you as well that the current implementation of KafkaContainer it made the extension and especially to have an improved SASL security on it super cumbersome.
> I did a few tries that look like the ones you shared in your previous link, and honestly not that happy with either of that.
> 
> For the purpose of here, I would rather go with having "something" useful and we can always come back to improve it. What do you think?

I totally agree. :-)

Reply:
@purbon, are you waiting for anything else from me? Am I supposed to resolve this conversation?
(Wish you a nice weekend!)